pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -1791,6 +1792,22 @@ Question.Decimal:
           type: array
           items:
             $ref: '#/Answer.Decimal'
+Question.Equation:
+  allOf:
+    - $ref: '#/Question'
+    - type: object
+      required:
+        - answers
+      properties:
+        scale:
+          type: integer
```

Comment:
It is not the scale that is the exponent. It is the number of digits after the comma to be displayed. Several lines above (in Decimal Question) it is defined as integer as well

Reply:
Maybe we can think about a better name to avoid names ambiguation?

Reply:
> It is not the scale that is the exponent.

The scale is the exponent- it's `unscaled * base ^ scale`, and we're fixed in base 10.

I agree that we need a new naming for this, though, as is a confusing mix. The usage here seems to be setting the maximum value for the scale/number of digits to the right of the point. What would you suggest?

Reply:
I guess I wasn't clear enough in my explanation.
We have **value** and **scale** in the definition of DecimalNumber type. They have an int64 type and are defined in another file. And those value and scale represent exactly what you described: `value * base ^ scale`.

The scale defined here is required by the front-end to cut off redundant digits after the comma. For instance, in the database, we store 123456789 as the value and 5 as the scale. If the scale defined here is not set, the front-end will render the number as 1234.56789. But if the scale is set to 2, for example, then the front-end will render the number as 1234.57 (or 1234.56) depending on rounding mode.

P.S. As for the exponent I mentioned. I was sure that in the English language **b** in expressions like **c = a ^ b** are is called the exponent. Google translate's it in this way as well. Could you please advise what is the right term I should use?

Reply:
The scale defined here might be renamed to **decimalPlacesCount**. I believe you can suggest a better name since you know what sounds better in English.

Reply:
Discussed personally.